src/app/shared/dso-page/dso-edit-menu/dso-edit-expandable-menu-section/dso-edit-menu-expandable-section.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="dso-button-menu mb-1" ngbDropdown container="body" placement="bottom-right">
+    <div class="d-flex flex-row flex-nowrap"
+         [ngbTooltip]="itemModel.text | translate">
+        <button class="btn btn-dark btn-sm" ngbDropdownToggle [disabled]="section.model?.disabled">
```

Comment:
Could we take this opportunity to add an `aria-label` to these buttons?  I ran a quick accessibility scan on this PR, and it's saying we are breaking the "Buttons must have discernable text" rule.  The fix is to ensure the button either has internal text, or has an `aria-label` or `title`.